src/WarcraftLegacies.Source/Quests/Sentinels/QuestScepterOfTheQueenSentinels.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.QuestSystem;
+using MacroTools.QuestSystem.UtilityStructs;
+using WarcraftLegacies.Source.Setup;
+using WarcraftLegacies.Source.Setup.Legends;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+namespace WarcraftLegacies.Source.Quests.Sentinels
+{
+    public class QuestScepterOfTheQueenSentinels : QuestData
+    {
+        public QuestScepterOfTheQueenSentinels(Rectangle area) : base("Return to the Fold", "Remnants of the ancient Highborne survive within the ruins of Dire Maul. If Stonemaul falls, it would be safe for them to come out.", "ReplaceableTextures\\CommandButtons\\BTNNagaWeaponUp2.blp")
+        {
+            HighBourneArea = area;
+            AddObjective(new ObjectiveLegendNotPermanentlyDead(LegendSentinels.legendFeathermoon));
+            AddObjective(new ObjectiveLegendDead(LegendWarsong.StonemaulKeep));
+            AddObjective(new ObjectiveAnyUnitInRect(Regions.HighBourne, "Dire Maul", true));
+            ResearchId = FourCC("R020");
```

Comment:
```suggestion
            ResearchId = Constants.UPGRADE_R02O_QUEST_COMPLETED_RETURN_TO_THE_FOLD_SENTINELS;
```
We use constants extracted from the map now. You can find these by typing in Constants.R02O into your IDE and letting it autocomplete for you.